### DISCUSSION TITLE
How to inline all CSS in a Next.js v13 project?

### DISCUSSION THREAD

**Discussion by lucgagan** [vercel/next.js#56409] [Help]
### Summary

I am using https://panda-css.com/ with Next.js v13 (app folder).

The pages load as expected, e.g. https://ray.run/blog/mastering-poms

But if you look at the rendered HTML, it links to CSS assets, e.g.

```
<link rel="stylesheet" href="/_next/static/css/3604df4459464cea.css" data-precedence="next"/>
```

I would like to inline this CSS in the response.

How do I do it?

### Example

https://ray.run/blog/mastering-poms

**Comment by langri-sha** [comment]
Has anyone tried using Webpack to post-process the production outputs and inline the stylesheets?
